src/operation/emulation_model.jl

```diff
@@ -235,28 +237,18 @@ function build_pre_step!(model::EmulationModel)
             @info "EmulationProblem status not BuildStatus.EMPTY. Resetting"
             reset!(model)
         end
-        # Initial time are set here because the information is specified in the
-        # Simulation Sequence object and not at the problem creation.
-        @info "Initializing Optimization Container for EmulationModel"
-
-        optimization_container_init!(
-            get_optimization_container(model),
-            get_network_formulation(get_template(model)),
-            get_system(model),
-        )
-        # Temporary while are able to switch from PJ to POI
-        get_optimization_container(model).built_for_recurrent_solves = true
-        @info "Initializing ModelStoreParams"
-        model_store_params_init!(model)
         set_status!(model, BuildStatus.IN_PROGRESS)
     end
     return
 end
 
+# TODO DT: should this be called build_impl!
+# Note that run! calls run_impl!
 function _build!(model::EmulationModel{<:EmulationProblem}, serialize::Bool)
     TimerOutputs.@timeit BUILD_PROBLEMS_TIMER "Problem $(get_name(model))" begin
         try
             build_pre_step!(model)
+            # TODO DT: Why is this called problem build?
```

Comment:
Potential naming consistency issue

Reply:
we need 2 methods generally speaking whether the build is called on its own or part of the simulation construction step. The reason for this was the ability to capture in the same logger the processes. We can review if needed.

Reply:
Change to `build_problem!`